libctru/include/3ds/services/am.h

```diff
@@ -25,41 +58,84 @@ void amExit(void);
 Handle *amGetSessionHandle(void);
 
 /**
- * @brief Gets the number of titles for a given mediatype.
- * @param mediatype Mediatype to get titles from.
+ * @brief Gets the number of titles for a given media type.
+ * @param mediatype Media type to get titles from.
  * @param count Pointer to write the title count to.
  */
-Result AM_GetTitleCount(u8 mediatype, u32 *count);
+Result AM_GetTitleCount(u32 *count, FS_MediaType mediatype);
```

Comment:
well, the thing is that it's breaking the API while there are other AM_functions that take the mediatype as first argument (AM_DeletePendingTitles,AM_DeletePendingTitle ...)
While this is only personal preference, I usually like to have the mode/type/enum first so that you know what value you will be getting from the following parameters.